src/patterns/start-pages/default.njk

```diff
@@ -0,0 +1,94 @@
+---
+layout: layout-example.njk
+stylesheets:
+- related-items.css
+---
+
+{% from "breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "button/macro.njk" import govukButton %}
+
+<div class="govuk-o-width-container">
```

Comment:
The indentation in this file is a bit funky but I'm not sure what the best way is to fix it. Maybe we should ignore it by now and find a way to prettify the HTML?

Reply:
The indentation funkiness is coming from the macros so not going to change for now.

Reply:
Funky.